stable/aws-vpc-cni/values.yaml

```diff
@@ -6,19 +6,43 @@
 # existing naming
 nameOverride: aws-node
 
+init:
+  image:
+    tag: v1.7.3
+    region: us-west-2
+    pullPolicy: Always
+    # Set to use custom image
+    # override: "repo/org/image:tag"
+  securityContext:
+    privileged: true
+
 image:
   region: us-west-2
-  tag: v1.6.3
+  tag: v1.7.3
   pullPolicy: Always
   # Set to use custom image
   # override: "repo/org/image:tag"
 
 # The CNI supports a number of environment variable settings
 # See https://github.com/aws/amazon-vpc-cni-k8s#cni-configuration-variables
 env:
+  ADDITIONAL_ENI_TAGS: "{}"
+  AWS_VPC_CNI_NODE_PORT_SUPPORT: "true"
+  AWS_VPC_ENI_MTU: "9001"
+  AWS_VPC_K8S_CNI_CONFIGURE_RPFILTER: "false"
+  AWS_VPC_K8S_CNI_CUSTOM_NETWORK_CFG: "false"
+  AWS_VPC_K8S_CNI_EXTERNALSNAT: "false"
+  AWS_VPC_K8S_CNI_LOG_FILE: "/host/var/log/aws-routed-eni/ipamd.log"
   AWS_VPC_K8S_CNI_LOGLEVEL: DEBUG
+  AWS_VPC_K8S_CNI_RANDOMIZESNAT: "prng"
   AWS_VPC_K8S_CNI_VETHPREFIX: eni
-  AWS_VPC_ENI_MTU: "9001"
+  AWS_VPC_K8S_PLUGIN_LOG_FILE: "/var/log/aws-routed-eni/plugin.log"
+  AWS_VPC_K8S_PLUGIN_LOG_LEVEL: DEBUG
+  DISABLE_INTROSPECTION: "false"
+  DISABLE_METRICS: "false"
+  DISABLE_TCP_EARLY_DEMUX: "false"
```

Comment:
Thanks a lot @shaunofneuron, this is awesome. Just one thing here, the `DISABLE_TCP_EARLY_DEMUX` is actually just used by the init container. It's only needed under some special circumstances when using the per pod security group feature.

https://github.com/aws/amazon-vpc-cni-k8s/blob/638c3ca21bde5b5f04a1c5aff228c472d81780f3/config/v1.7/aws-k8s-cni.yaml#L195-L199

Reply:
Totally missed that, good catch

Reply:
Exposed init env and shifted that var into it.

Reply:
Exposed init env and moved var into that.